packages/apiclient-services-subscribewithgoogle/SubscribewithGoogle.php

```diff
@@ -0,0 +1,461 @@
+<?php
```

Comment:
Note the contents of this file are entirely generated and should be treated as an external dependency. It's inclusion here is temporary until it is available as part of the `apiclient-services` package.